lib/rubocop/cop/root_cops/retry_on_warning.rb

```diff
@@ -0,0 +1,18 @@
+require "pry"
```

Comment:
Why the require pry?  If this is required shouldn't it be listed as a dependency in the gemspec?

I call this in non-rails apps that don't have pry in the Gemfile so this causes rubocop to throw exceptions.

Reply:
https://github.com/Root-App/root-ruby-style/pull/15